gapir/replay_service/service.proto

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package replay_service;
```

Comment:
Sorry to go back to this again, but what was the original issue with this being called `service`? Did you say there was a collision between this and gapis service?

Reply:
`package service` => name conflict with gapis service
`package gapir` and `service Service` => C++ won't compile, and results into weird folder naming like `gapir/gapir/service.proto`, that `gapir/gapir` hierarchy is terrible.

So to keep the folder's name and the package name the same (so the generated code location will be consistent), renamed the package to replay_service. I think this makes sense, as replay_service is part of the functionalities of GAPIR.

Reply:
When you say it collides with gapis service, when? I assume gapir has no reason to import the gapis protos. So is this a collision in GAPIS at runtime?

Reply:
And the `RegisterType`:
https://github.com/golang/protobuf/blob/master/proto/properties.go#L482